docs/source/tutorial/trackers/using_mlflow.rst

```diff
@@ -78,5 +83,13 @@ different sub-experiments together using the ``experiment_id`` keyword argument
         ),
     )
 
+Adding Tags
```

Comment:
@sunny1401 I've added an outline of what would be helpful for a user who isn't already familiar with this. Please answer the two questions in prose, and create a minimal runnable code example to go with it (please try to avoid cryptic variable names and undefined variables).

It's not good to have the tags example at the top, because the user should learn about one feature at a time in the tutorial. The first example is a bare bones minimum example with no optional features. You can see below there are dedicated sections (okay, just one before now) on each optional feature

Reply:
yep doing that.. Sorry - I din know how exactly to follow so I just added above.